weatherman.py

```diff
@@ -0,0 +1,80 @@
+"""
+Generating reports of temperature from given text files.
+"""
+
+import os
+import argparse
+import sys
```

Comment:
Fix import order (PEP-8)

Reply:
One source that was explaining PEP-8 mentioned that they need to be in alphabetical order. Did not find this in PEP-8 myself. Couldn't find argparse in Standard library imports, and wasn't sure what type this is exactly. But I have changed it a little, please check this and talk to me about this.

Reply:
It is standard library import. So just use alphabetical order.

Reply:
Done.